alonzo/impl/src/Cardano/Ledger/Alonzo/PlutusScriptApi.hs

```diff
@@ -188,6 +190,7 @@ collectTwoPhaseScriptInputs pp tx utxo =
         Just script -> Right script
         Nothing -> Left (NoWitness hash)
     apply cost (sp, d, eu) script = (script, d : (valContext txinfo sp) : (getData tx utxo sp), eu, cost)
+    -- wrong order arguments, should be (context, datum, redeemer)
```

Comment:
let's ask @michaelpj what order plutus is expecting

Reply:
order is datum, redeemer, context

Reply:
fixed in https://github.com/input-output-hk/cardano-ledger-specs/pull/2259